src/providers/steem/dsteem.js

```diff
@@ -343,6 +353,8 @@ export const getPost = async (author, permlink, currentUserName = null) => {
   try {
     const post = await client.database.call('get_content', [author, permlink]);
 
+    const ugur = await getState(`/${post.category}/@${post.author}/${post.permlink}`);
+    console.log(ugur);
```

Comment:
ugur? :smile:

Reply:
Sorry I forgot to remove god mode variable 🤣